src/Interfaces/Options/Background/IBackground.ts

```diff
@@ -0,0 +1,11 @@
+import type { IColor } from "../Particles/IColor";
+import type { IOptionLoader } from "../IOptionLoader";
+
+export interface IBackground extends IOptionLoader<IBackground> {
+    color?: IColor | string;
+    image?: string;
+    position?: string;
+    repeat?: string;
+    size?: string;
+    opacity?: number;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5274115)